k9mail/src/main/java/com/fsck/k9/helper/HtmlSanitizer.java

```diff
@@ -0,0 +1,47 @@
+package com.fsck.k9.helper;
+
+
+import org.htmlcleaner.CleanerProperties;
+import org.htmlcleaner.HtmlCleaner;
+import org.htmlcleaner.SimpleHtmlSerializer;
+import org.htmlcleaner.TagNode;
+
+
+public class HtmlSanitizer {
+    private HtmlSanitizer() {}
+
+    public static String sanitize(String html) {
+        HtmlCleaner cleaner = new HtmlCleaner();
```

Comment:
They claim it's thread-safe. Do you have evidence that suggests the version we're using isn't?
http://htmlcleaner.sourceforge.net/javause.php#example2